debian/source/format

```diff
@@ -0,0 +1 @@
+3.0 (quilt)
```

Comment:
curious what does that mean ?

Reply:
https://www.debian.org/doc/manuals/maint-guide/dother.en.html#sourcef
It seems that quilt is the correct option